src/main/java/it/gov/pagopa/pu/classification/service/NamespaceFilter.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.pu.classification.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.AttributesImpl;
+import org.xml.sax.helpers.XMLFilterImpl;
+
+
+@Slf4j
+public class NamespaceFilter extends XMLFilterImpl {
+
+  private static final String NAMESPACE = "http://www.regione.veneto.it/schemas/2012/Pagamenti/Ente/";
```

Comment:
This could be generic? From a generic `NamespaceFilter` I would expect a generic (dummy) namespace
```suggestion
  private static final String NAMESPACE = "http://www.piattaforma-unitaria.spac.it";
```

otherwise the applied namespace should be a parameter provided when creating the class